src/Controller/FeatureController.php

```diff
@@ -106,6 +116,19 @@ public function edit(Request $request, Feature $feature, FeatureManager $manager
         return $this->render('feature/edit.html.twig', ['feature' => $feature, 'form' => $form]);
     }
 
+    #[IsGranted(Permissions::DELETE, 'feature')]
+    #[Route(path: '/delete/{id}', name: 'feature_delete', methods: ['POST'])]
+    public function delete(Request $request, Feature $feature, UserRequestManager $manager): Response
+    {
+        $csrfTokenName = sprintf('delete%d', $feature->getId());
+        if ($this->isCsrfTokenValid($csrfTokenName, (string) $request->request->get('_token', ''))) {
```

Comment:
I think instead of string casting, you canuse getAlpha('_token'), then no need to specify default as it is '' bu default